common/on-chip-calib.cpp

```diff
@@ -1139,6 +1128,18 @@ namespace rs2
         {
             try
             {
```

Comment:
Let's talk about it?
I want to understand why it's on the else, hard to understand with this huge code :)